test/node-examples.js

```diff
@@ -137,9 +139,17 @@ async function runTestExample(folder) {
   exampleDir = EXAMPLES_GET_STARTED_DIR;
   await runTestExample('pure-js');
   await runTestExample('pure-js-without-map');
-  // await runTestExample('react-webpack');
+  // await runTestExample('react-browserify');
+  await runTestExample('react-webpack-2');
   await runTestExample('react-without-map');
 
+  exampleDir = EXAMPLES_JSON_BROWSER_DIR;
+  await runTestExample('pure-js-app');
```

Comment:
We are starting to run into name collisions (`pure-js` vs `pure-js-app`). The naming convention of golden images needs to be updated to include directory names.